Formula/uthash.rb

```diff
@@ -0,0 +1,54 @@
+class Uthash < Formula
+  desc "C macros for hash tables and more"
+  homepage "https://troydhanson.github.io/uthash/"
+  url "https://github.com/troydhanson/uthash/archive/refs/tags/v2.3.0.tar.gz"
+  sha256 "e10382ab75518bad8319eb922ad04f907cb20cccb451a3aa980c9d005e661acc"
+  license "BSD-1-Clause"
+  head "https://github.com/troydhanson/uthash.git", branch: "master"
+
+  def install
+    include.install Dir["src/*.h"]
```

Comment:
```suggestion
    include.install buildpath.glob("src/*.h")
```
Since the `*.h` files aren't directories (which is what `Dir` kind of suggests).